### PR TITLE
updated Mime.types with new MIME types

### PR DESCRIPTION
**Pull request by limenet**
As per issue #15 
I hope I didn't mess up anything, since there are a **lot** of changes I can't go through them one-by-one but after a few checks it looks fine.

### CODE REVIEW

**Comment by Hywan** [comment]
@limenet Could you please explain your process to update the `Mime.types` file?

File: Mime.types [hunk 1]
```diff
@@ -535,74 +1436,164 @@ multipart/related
...
 text/sgml					sgml sgm
 text/t140
 text/tab-separated-values	tsv
-text/uri-list
-text/vcard					vcf vcard
+text/troff					t tr roff man me ms
+text/turtle					ttl
+text/ulpfec
+text/uri-list					uri uris urls
+text/vcard					vcard
```
Comment (Hywan) [inline Mime.types]: `vcf` is also a format for `text/vcard`.

File: Mime.types [hunk 2]
```diff
@@ -535,74 +1436,164 @@ multipart/related
...
+text/x-fortran					f for f77 f90
+text/x-java-source				java
+text/x-nfo					nfo
+text/x-opml					opml
+text/x-pascal					p pas
+text/x-setext					etx
+text/x-sfv					sfv
+text/x-uuencode					uu
+text/x-vcalendar				vcs
+text/x-vcard					vcf
```
Comment (Hywan) [inline Mime.types]: Hmm, `vcf` is here.

**Comment by Hywan** [comment]
Sounds good. Just check is `vcf`is really `text/x-vcard`. I am not sure.

**Comment by Hywan** [comment]
Ok, `text/x-vcard` is deprecated: http://tools.ietf.org/html/rfc6350#section-10.1.

**Comment by Hywan** [comment]
So it's a :+1: for me.

**Comment by Hywan** [comment]
Welcome on board: https://github.com/hoaproject/Central/blob/master/CONTRIBUTOR.md ;-).

**Comment by limenet** [comment]
Awesome, thanks! :smiley:

**Pull request merged**